tests/upgrades/test_contentview.py

```diff
@@ -279,3 +297,214 @@ def test_post_user_scenario_bug_1429201(self):
         self.assertIsNotNone(container_ids)
         self.assertIn('Error', list(result_fail.values())[0])
         self.assertIn('Complete', list(result_pass.values())[0])
+
+
+class CVUpgradeScenario(CLITestCase):
```

Comment:
```suggestion
class Scenario_contentview_upgrade(CLITestCase):
```

Reply:
@vijay8451 we can not give `'Scenario_contentview_upgrade'` as class name because it's not according to class name convention may be we should use `'ScenarioContentViewUpgrade'`.

Reply:
@jameerpathan111 we are following the same way naming for all scenarios test cases around upgrade. also naming not ugly as per PEP8 too afaik.

here is i.e.
https://github.com/SatelliteQE/robottelo/blob/master/tests/upgrades/test_subscription.py#L35
https://github.com/SatelliteQE/robottelo/blob/master/tests/upgrades/test_capsule.py#L32